docker/docker-compose/docker-compose.yml

```diff
@@ -10,13 +10,10 @@ services:
       - 8030:8030
       - 9020:9020
       - 9030:9030
-    # volumes:
-    #   - ../../conf/fe.conf:/opt/starrocks/fe/conf/fe.conf
-    healthcheck:
-      test: ["CMD", "curl", "-f", "http://localhost:9030"]
-      interval: 5s
-      timeout: 5s
-      retries: 30
+#    volumes:
+#      - path/of/host/fe.conf:/opt/starrocks/fe/conf/fe.conf
+#      - path/of/host/fe/meta:/opt/starrocks/fe/meta
```

Comment:
point that the path should exist in current dir.

Reply:
Done